### ISSUE TITLE
OAV grid detection

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/python-artemis#402]
The GDA grid detection on the OAV image creates a grid with the max range e.g. https://ispyb.diamond.ac.uk/dc/visit/cm31105-5/id/9330896. However, in OAV centring the pin width is calculated using the first inflection point of the pin. If we used this technique in Artemis to find the grid area we could get away with smaller grids.

## Acceptance criteria
* Discuss with @ndg63276 and @npat113 if this is sensible
* If it is, try it and evaluate time savings compared to potential accuracy issues

**Comment by npat113** [comment]
you're assuming the crystal is in the loop. Rookie error

**Comment by npat113** [comment]
some form of x restriction from tip would be sensible though, and necessary if we were to move to grid setting at zoom 1 as part of the initial centring  after robot load (which I would like to try).

**Comment by npat113** [comment]
![Screenshot 2022-11-28 at 12 21 16](https://user-images.githubusercontent.com/106971253/204276600-95dbb7a1-913a-4127-8bd4-3ece7dacc49d.png)

**Comment by ndg63276** [comment]
Yeah we should definitely discuss what a sensible max width of the grid should be. At the moment we do "to the edge of the screen at zoom level 7.5" which is completely arbitrary (and about 650 microns).

**Comment by DominicOram** [comment]
Thanks both. We should discuss when we get closer to thinking about it
